src/plugins/LADTable/plugin.js

```diff
@@ -0,0 +1,56 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2018, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+define([
+    './LADTableViewProvider',
+    './LADTableSummaryViewProvider'
+], function (
+    LADTableViewProvider,
+    LADTableSummaryViewProvider
+) {
+    return function plugin() {
+        return function install(openmct) {
+            openmct.objectViews.addProvider(new LADTableViewProvider(openmct));
+            openmct.objectViews.addProvider(new LADTableSummaryViewProvider(openmct));
+
+            openmct.types.addType('LADTable', {
+                name: "LAD Table",
+                creatable: true,
+                description: "A Latest Available Data tabular view in which each row displays the values for one or more contained telemetry objects.",
+                cssClass: 'icon-tabular-lad',
+                initialize(domainObject) {
+                    domainObject.composition = [];
+                }
+            });
+
+            openmct.types.addType('LADTableSummary', {
+                name: "LAD Table Summary",
```

Comment:
How about "LAD Table Set" instead of "LAD Table Summary"?  As in, "a set of LAD tables".

The idea is that a lad table set should include a separator (the name of the lad table) between each set of channels).